backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
```

Comment:
We can change it to `test_type`? Not sure.

Reply:
I think we should converge all the nomenclature.

Reply:
We have updated the glossary to say "test_methodology" as this table is named here.